app/views/journeys/expenses/officeSupplies/OfficeSuppliesDisallowableAmountView.scala.html

```diff
@@ -16,32 +16,37 @@
 
 @import viewmodels.InputWidth._
 @import controllers.journeys.expenses.officeSupplies.routes.OfficeSuppliesDisallowableAmountController
+@import views.html.templates.helpers.Heading
 
 @this(
     layout: templates.Layout,
     formHelper: FormWithCSRF,
     govukErrorSummary: GovukErrorSummary,
     govukInput: GovukInput,
-    govukButton: GovukButton
+    govukButton: GovukButton,
+    heading: Heading
 )
 
-@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], mode: Mode, taxYear: Int, businessId: String, authUserType: String, allowableAmount: BigDecimal)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = title(form, messages("officeSuppliesDisallowableAmount.title"))) {
 
-    @formHelper(action = OfficeSuppliesDisallowableAmountController.onSubmit(mode), Symbol("autoComplete") -> "off") {
+    @formHelper(action = OfficeSuppliesDisallowableAmountController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
 
         @if(form.errors.nonEmpty) {
             @govukErrorSummary(ErrorSummaryViewModel(form))
         }
 
+        @heading(messages(s"officeSuppliesDisallowableAmount.title.$authUserType", allowableAmount.toString), None, extraClasses = "govuk-!-margin-bottom-3")
+
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(messages("officeSuppliesDisallowableAmount.heading")).asPageHeading()
+                label = LabelViewModel("").asPageHeading()
```

Comment:
if there's no label it doesn't need .asPageHeading(). But the error should encompass the title on this page so think the heading needs to be the label, not in the @heading() on line 40

Reply:
Good spot.